operations/services.md

```diff
@@ -207,10 +217,23 @@ Returns selected availability and occupancy metrics of a bookable service for a
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `ServiceId` | string | required | Unique identifier of the [Service](#service) whose availability should be returned. |
-| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit] (../guidelines/time-units.md), in UTC timezone ISO 8601 format. |
-| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit] (../guidelines/time-units.md), in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's [time unit](../guidelines/time-units.md): 367 hours if hours, 367 days if days, or 24 months if months. |
-| `Metrics` | array of [Service availability metrics](#service-availability-metrics) | required | Set of [Service availability metrics](#service-availability-metrics) to be returned. |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) whose availability should be returned. |
+| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit](../guidelines/time-units.md), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit](../guidelines/time-units.md), in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's [time unit](../guidelines/time-units.md): 367 hours if hours, 367 days if days, or 24 months if months. |
+| `Metrics` | array of [Service availability metrics](#service-availability-metrics) | required | Set of [Service availability metrics](services.md#service-availability-metrics) to be returned. |
+
+#### Service availability metrics
+
+* `OutOfOrderBlocks`
+* `PublicAvailabilityAdjustment`
+* `OtherServiceReservationCount`
+* `Occupied`
+* `ConfirmedReservations`
+* `OptionalReservations`
+* `BlockAvailability`
+* `AllocatedBlockAvailability`
+* `UsableResources`
+* `ActiveResources`
```

Comment:
Assuming it works correctly (since these are dictionary keys), you should be able to bring these descriptions back by putting `[Description]` on each enum value.